pkg/ccl/changefeedccl/compression.go

```diff
@@ -17,24 +18,131 @@ import (
 	"github.com/klauspost/pgzip"
 )
 
-var useFastGzip = settings.RegisterBoolSetting(
-	settings.ApplicationLevel,
-	"changefeed.fast_gzip.enabled",
-	"use fast gzip implementation",
-	metamorphic.ConstantWithTestBool(
-		"changefeed.fast_gzip.enabled", true,
-	),
-	settings.WithPublic)
+var (
+	useFastGzip = settings.RegisterBoolSetting(
```

Comment:
nit: don't group this together with the sync pools

Reply:
Fixed